rust/tracing/src/init_tracer.rs

```diff
@@ -38,6 +38,7 @@ pub fn init_global_filter_layer() -> Box<dyn Layer<Registry> + Send + Sync> {
                 "wal3",
                 "worker",
                 "garbage_collector",
+                "axum",
```

Comment:
will this result in a lot of span spam? if not, maybe worth commenting why

Reply:
Oops this is an accidental leave in. Thanks for the catch